proto/quotacontrol.ridl

```diff
@@ -18,7 +18,7 @@ enum Service: uint16
 struct Limit
   # The service-specific limits for the project.
   - serviceLimit: map<string,ServiceLimit>
-    + go.field.type = map[Service]ServiceLimit
+    + go.field.type = map[string]ServiceLimit
```

Comment:
this looks good to me -- an old service will be able to update limits for all (new) services